src/rcall.jl

```diff
@@ -67,3 +67,28 @@ function sexp(tree::NamedTree)
     unprotect(1)
     return sobj
 end
+
+eltype(::Type{RClass{:phylo}}, s::Ptr{VecSxp}) = Phylo
+function rcopy(::Type{RClass{:phylo}}, s::Ptr{VecSxp})
```

Comment:
1. The best practice is to first define
the explicit copier `rcopy(::Type{Phylo}, s::Ptr{VecSxp})`. It will also define a converter `convert(::Type{Phylo}, r::RObject)`

2. `eltype` is only needed if you want to define `rcopy(Array, r)`.

3. I would just define `rcopy(::Type{RClass{:phylo}}, s::Ptr{VecSxp}) = rcopy(Phylo, s)`